INFO     root:Log.py:79 [INFO 2019-11-01 17:17:56]初始化关联文件
DEBUG    root:Log.py:73 [DEBUG 2019-11-01 17:17:56]读取初始关联文件内容：   ItemsView(<Section: test_data>)
DEBUG    root:Log.py:73 [DEBUG 2019-11-01 17:17:56]初始关联内容数据处理后：   {'name': '10', 'type': 'Web', 'version': 'Yv3LA2jl1V', 'description': 'api_auto_3BoW6XOQkd', 'project_id': '91ab8e7170944a1db9a4405f83ba8680', 'token_id': '8fec885ed3e843b8b8276e4bd7ca0635', 'project_name': 'Xj0f7nwNxhJHps41LASr', 'network_name': 'r9blde2nIoFZuQJqUk5X_netwrok', 'subnet_name': 'ykY6E5vUPXWjHnqgKc1r_subnet', 'network_id': '4ceca06f-1552-4d6c-8a27-7082e59df91b', 'subnet_id': '7627b375-7250-4756-8cc2-f519dc952dd4', 'image_id': 'b4e9e907-686e-440d-bc1a-d73a8decfa54', 'flavor_id': 'flavor-1-1-1', 'server_name': 'wuzs_wKifYhI4tqTAazmZWr8d', 'server_id': 'bba73f7e-5288-4814-bca0-9a0f408c7b2e', 'project_token_name': 't7oTIRGY3uLPHBqmnS4h', 'admin_id': '0ec82b7380cb4d0c80058501938cdf48', 'admin_role_id': 'cc33f2c3a47247329d0ee7043c4f475f', 'volume_name': 'volume_Xp3mBtFZ1YfUOA9olIRM', 'volume_id': 'b1d5cbd5-1cb7-45ce-81cb-d43009ff12a0', 'server_snap_name': 'snap_8JkrX4yjSduLBPHcmTxC', 'server_snap_id': '', 'volume_snapshot_id': '8929f975-0d7f-4ab6-8fe2-fa92f8b51de7', 'network_name_for_check': 'OHzX9BxtmeJhDMEpjNi1_netwrok'}
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]====================================================================================================
DEBUG    root:Log.py:73 [DEBUG 2019-11-01 17:17:58]请求头处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': '8fec885ed3e843b8b8276e4bd7ca0635'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-01 17:17:58]请求参数处理结果：  {'Content-Type': 'application/json', 'X-Auth-Token': '8fec885ed3e843b8b8276e4bd7ca0635'}
DEBUG    root:Log.py:73 [DEBUG 2019-11-01 17:17:58]host处理结果：  192.168.54.120
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]请求接口：删除网络后查看网络
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]请求地址：http://192.168.54.120:9696/v2.0/networks/4ceca06f-1552-4d6c-8a27-7082e59df91b
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]请求头: {'Content-Type': 'application/json', 'X-Auth-Token': '8fec885ed3e843b8b8276e4bd7ca0635'}
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]请求参数: {'subnet': {'admin_state_up': True}}
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]GET请求接口
DEBUG    urllib3.connectionpool:connectionpool.py:205 Starting new HTTP connection (1): 192.168.54.120:9696
DEBUG    urllib3.connectionpool:connectionpool.py:393 http://192.168.54.120:9696 "GET /v2.0/networks/4ceca06f-1552-4d6c-8a27-7082e59df91b?subnet=admin_state_up HTTP/1.1" 404 137
INFO     root:Log.py:79 [INFO 2019-11-01 17:17:58]接口请求结果：
                         (404, {'NeutronError': {'message': 'Network 4ceca06f-1552-4d6c-8a27-7082e59df91b could not be found', 'type': 'NetworkNotFound', 'detail': ''}})